### PR TITLE
treewide: Drop infinisil as maintainer from most packages

### PR DESCRIPTION
**Pull request by infinisil**
## Description of changes

I'm not going anywhere, I'm just focusing my energy on other issues, and getting pinged as a maintainer for packages is a bit distracting (also I'm not using most of these packages anyways!).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by felixsinger — APPROVED** [review]
Ack for Mumble

File: pkgs/tools/misc/ideviceinstaller/default.nix [hunk 1]
```diff
@@ -49,7 +49,7 @@ stdenv.mkDerivation rec {
     '';
     license = licenses.gpl2Plus;
     platforms = platforms.unix;
-    maintainers = with maintainers; [ aristid infinisil ];
+    maintainers = [ aristid ];
```
Comment (wegank) [inline pkgs/tools/misc/ideviceinstaller/default.nix:52]:
```suggestion
    maintainers = with maintainers; [ aristid ];
```

Comment (infinisil) [reply pkgs/tools/misc/ideviceinstaller/default.nix]: Oof this is the second instance of that, fixed!

Comment (infinisil) [reply pkgs/tools/misc/ideviceinstaller/default.nix]: And had to fix another :sweat_smile:

**Review by wegank — APPROVED** [review]

**Pull request merged**